src/FakeItEasy/Configuration/ArgumentCollection.cs

```diff
@@ -68,6 +69,8 @@ public int Count
         /// </summary>
         public IEnumerable<string> ArgumentNames { get; }
 
+        internal MethodInfo Method { get; }

```

Comment:
> We could, but it would make the tests harder to write, since they would need to provide a `MethodInfo`

That's true, but I'd rather tests were a little harder to write than the doubt of the extra constructor. Also, I wonder how many of the tests are [Gulliver's Travels Tests](http://dhickey-ie-archive.azurewebsites.net/post/2014/03/03/gulliver-s-travels-tests.aspx)?

(I'm happy to investigate, but it'll probably be slower than you'd like.)

> OK, but how do I get to the `IFakeObjectCall`? It's not exposed in any way by `IArgumentValidationConfiguration<TInterface>`...

It's true. I told you I hadn't tried it! :grinning:
I don't have an answer. I feel like there should be a way, but I'm coming up short right now.